configs/orchestrator/templates/orchestrator-config.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: orchestrator-config
```

Comment:
Is there a reason to keep this config map at all? It seems like the only thing it ever did was holding the CT version?

Reply:
I need to make the fix in the golang orchestrator application and then this configmap won't be needed. This is a temporary change.

Reply:
Got it. LGTM then.